src/game-of-life/setup.md

```diff
@@ -117,9 +117,29 @@ greet("Rust and WebAssembly");
 ### `src/lib.rs`
 
 The `src/lib.rs` file is the root of the Rust crate that we are compiling to
-WebAssembly. It uses `wasm_bindgen` to interface with JavaScript. It imports the
-`window.alert` JavaScript function, and exports the `greet` Rust function, which
-takes a `name` parameter and alerts a greeting message.
+WebAssembly. Let's step through this line by line.
+
+First up is the list of nightly features. We're enabling this for the whole
+crate. What this means is that we will later tag code with an attribute and
+this will allow Rust to generate code that we don't have to write by hand. In
+our case it'll use `wasm-bindgen`. It should be noted that `#![feature(...)]`
+implies using the nightly compiler. This gated feature will hopefully be
+stabilized and landed soon so that you won't need it!
+
+`wasm-bindgen` knows how to make code that works well with wasm so we don't
+have to worry about it too much and just write Rust code for the most part. If
+you want to know the full extent of its capabilities check out the README on
+its repo which can be found
+[here](https://github.com/alexcrichton/wasm-bindgen). For our purposes we need
+to know that if we want functions to work with wasm easily we'll need it.
```

Comment:
This last sentence isn't very clear to me.

In fact, I think we can replace these first two paragraphs with simply:

```md
> We will use [wasm-bindgen](https://github.com/rustwasm/wasm-bindgen) to
> communicate between Rust/wasm and JavaScript. `wasm-bindgen` currently
> relies on nightly-only features (`proc_macro`, etc...) that we must opt
> into using.
```

I want to keep the intro very short and to the point because people bounce off of walls of text, especially when first getting started before they are hooked.

Reply:
Ack.